pkgs/development/python-modules/pyside6/default.nix

```diff
@@ -27,6 +28,8 @@ stdenv.mkDerivation rec {
     cmake
     ninja
     python
+  ] ++ lib.optionals stdenv.isDarwin [
+    moveBuildTree
   ];
 
   buildInputs = with qt6; [
```

Comment:
If we're not building the optional Qt modules anymore, then it's probably worth changing them to be Linux only `buildInputs`

Reply:
Done.